netaddr.go

```diff
@@ -780,6 +788,43 @@ func (ip IP) string6() string {
 	return string(ret)
 }
 
+// StringExpanded returns the string form of the IP address ip, but IPv6
+// addresses are expanded with leading zeroes and no "::" compression. For
+// example, "2001:db8::1" becomes "2001:0db8:0000:0000:0000:0000:0000:0001".
+//
+// IPv4 and invalid IP addresses are formatted the same as when using IP.String.
+// See the documentation of IP.String for more details.
+func (ip IP) StringExpanded() string {
+	// Invalid and IPv4 are handled the same as in String.
+	switch ip.z {
+	case z0, z4:
+		return ip.String()
+	}
+
+	// Allocate enough space for the longest possible IPv6 address plus the
```

Comment:
This is no longer the longest possible IPv6 address, it's the exact size of the output. Rename variable to `sz` and remove this comment block, once the variable name references size it's sufficiently evident what's going on.